tests/framework/ROM/SKLearn/gold/data/outMLPRegressorModel.xml

```diff
@@ -0,0 +1,16 @@
+<DataObjectMetadata name="modelOut">
+  <modelUnderTest type="Static">
+    <ROM>
+      <type>SupervisedLearning</type>
+      <coefs_>[[ 0.57866553 -0.14216937 -0.49877795  0.00708437 -0.69097522]
+ [ 0.19793753 -0.72966728  0.16291997 -0.59780582  0.19012253]],[[ 0.55827053 -0.5684078 ]
+ [ 0.38937298  0.55169136]
+ [-0.47988497  0.21313262]
+ [-0.4190013  -0.79921692]
+ [-0.45824578 -0.41478894]],[[ 0.38117052]
+ [ 0.65072803]]</coefs_>
+      <intercepts_>[ 0.42858406 -0.61337638 -0.03455315  0.65379859  1.05020687],[-0.52096305  0.32170952],[ 0.80302545]</intercepts_>
```

Comment:
hm, yeah, especially since whitespace including newlines is assumed irrelevant in some XML readers. This is probably a fine start, although will take some nitpicking to parse on the back end.